src/components/WeeklyCalendar/helpers/getCurrentWeek.js

```diff
@@ -0,0 +1,21 @@
+import getFirstDayOfWeek from './getFirstDayOfWeek';
+import { normalizeDate, isDateBelowLimit, isDateBeyondLimit } from '../../Calendar/helpers';
+
+export default function getCurrentWeek(currentWeek, minDate, maxDate) {
+    const week = getFirstDayOfWeek(normalizeDate(currentWeek));
+    week.setHours(0, 0, 0, 0);
+
+    if (minDate) {
+        const minWeek = getFirstDayOfWeek(normalizeDate(minDate));
+        minWeek.setHours(0, 0, 0, 0);
+        if (isDateBelowLimit(week, minWeek)) return minWeek;
```

Comment:
I guess what you want to do here can be resolve using Math.max function

```
return Math.max(week, minWeek);
```
if minWeek wasn't passed then will fallback into `-Infinity`